internal/bus/chainsubscriber.go

```diff
@@ -269,10 +279,12 @@ func (s *chainSubscriber) revertChainUpdate(tx sql.ChainUpdateTx, cru chain.Reve
 
 	// v2 contracts
 	cus = cus[:0]
-	var revertedContracts []types.FileContractID
+	var revertedContracts []types.V2FileContractElement
 	cru.ForEachV2FileContractElement(func(fce types.V2FileContractElement, created bool, rev *types.V2FileContractElement, res types.V2FileContractResolutionType) {
 		if created {
```

Comment:
@n8maninger wasn't sure about this one. Thinking about it maybe this is redundant anyway since we should apply a block whenever we revert one so we'd override the element anyway.

Reply:
It would probably be fine, but better to always cleanly revert imo.